plutus-core/untyped-plutus-core/src/UntypedPlutusCore/Core/Instance/Flat.hs

```diff
@@ -232,20 +233,3 @@ instance ( Closed uni
     encode (Program ann v t) = encode ann <> encode v <> encode t
     decode = Program <$> decode <*> decode <*> decode
     size (Program a v t) n = n + getSize a + getSize v + getSize t
```

Comment:
I wonder if this is:

```
deriving anyclass instance
         ( Closed uni
         , uni `Everywhere` Flat
         , PrettyPlc (Term name uni fun ann)
         , Flat fun
         , Flat ann
         , Flat name
         , Flat (Binder name)
         ) => Flat (Program name uni fun ann)
```

Reply:
Yeah, I think we just didn't want to use any derived instances so we could be 100% sure we control the instance and it won't change if the library changes.